.drone.yml

```diff
@@ -289,6 +289,45 @@ volumes:
   - name: dockersock
     temp: {}
 
+---
+kind: pipeline
+type: kubernetes
+name: tag-build-plugins-helm
+
+trigger:
+  event:
+    - tag
+  ref:
+    include:
+      - refs/tags/teleport-jira-v*
+      - refs/tags/teleport-mattermost-v*
+      - refs/tags/teleport-pagerduty-v*
+      - refs/tags/teleport-slack-v*
+      - refs/tags/teleport-email-v*
```

Comment:
That will be handled separately, although it might fit in perfectly with the current pipeline. I haven't investigated it and wanted to move forward with email-handler first. 

I'd like to if possible address additional drone pipeline needs with the PRs that adds those charts. Which I plan on contributing to as well. Although, the promotion step will be handled outside of those PRs.

Reply:
> I'd like to if possible address additional drone pipeline needs with the PRs that adds those charts. Which I plan on contributing to as well. Although, the promotion step will be handled outside of those PRs.

Sounds good to me!